admin/class-export.php

```diff
@@ -125,7 +125,8 @@ private function write_opt_group( $opt_group ) {
 			if ( is_array( $elem ) ) {
 				$count = count( $elem );
 				for ( $i = 0; $i < $count; $i++ ) {
-					$this->write_setting( $key . '[]', $elem[ $i ] );
+					$elem_check = isset( $elem[ $i ] ) ? $elem[ $i ] : '';
+					$this->write_setting( $key . '[]', $elem_check );
```

Comment:
With this handling, we have a difference between the output this PR gives versus to what we have in our production versions.

Namely, there are some options that are associative arrays, for example `company_logo_meta` that with this PR, we're going to get
```company_logo_meta[] = ""``` 
and we used to get 
```company_logo_meta[] = 	```

While this is an improvement, I think we either have to:
* Properly fix this - because with our current fix, we don't import the `company_logo_meta` setting or the other settings that have assosiative arrays as values
  * If we want to do that, maybe we should consider doing it for 20.4 instead, since it increases the scope
* Not fix this yet, but do change our current fix, so that the output of this PR matches the output the output of our production version

Reply:
Changed it to `null`